src/components/description-list/_description-list.scss

```diff
@@ -1,6 +1,14 @@
 .ons-description-list {
     &__items {
         margin: 0 0 2rem;
+
+        &.ons-description-list--horizontal {
```

Comment:
This class is set in the html at the same level as the items class. So it should probably be structured where they are are the same level and not inside eachother